x-pack/platform/plugins/shared/stack_connectors/public/common/config_service.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ConfigSchema } from '../../server/config';
+
+type Config = Omit<ConfigSchema, 'enableExperimental'>;
+
+export class ConfigService {
```

Comment:
I did not want to mangle with the existing experimental features that do not support offering-based config values, so I created this config service. I will remove it once the feature flag can be remove.